boards/BMS/Src/stm32f3xx_hal_msp.c

```diff
@@ -108,11 +108,12 @@ void HAL_ADC_MspInit(ADC_HandleTypeDef *hadc)
         __HAL_RCC_GPIOA_CLK_ENABLE();
         /**ADC1 GPIO Configuration
         PA2     ------> ADC1_IN3
+        PA3     ------> ADC1_IN4
         */
-        GPIO_InitStruct.Pin  = TS_VSENSE_P_Pin;
+        GPIO_InitStruct.Pin  = TS_VSENSE_P_Pin | TS_VSENSE_N_Pin;
         GPIO_InitStruct.Mode = GPIO_MODE_ANALOG;
         GPIO_InitStruct.Pull = GPIO_NOPULL;
-        HAL_GPIO_Init(TS_VSENSE_P_GPIO_Port, &GPIO_InitStruct);
+        HAL_GPIO_Init(GPIOA, &GPIO_InitStruct);
```

Comment:
dont forget to name this to TS_VSENSE_P on cube

Reply:
not sure where this is. TS_VSENSE is no longer on a GPIO pin. Its on ADC1 now

Reply:
This should be a user constant specified in cubeMX